dask/array/core.py

```diff
@@ -76,6 +76,15 @@
 tensordot_lookup.register((object, np.ndarray), np.tensordot)
 einsum_lookup.register((object, np.ndarray), np.einsum)
 
+unknown_chunk_message = (
+    "\n\n"
+    "A possible solution: "
+    "https://docs.dask.org/en/latest/array-chunks.html#unknown-chunks\n"
```

Comment:
Anybody have objections to including URLs in the error message? I personally don't have a problem with it.

Reply:
No objection from me

Reply:
It's also in the chunks error message:

https://github.com/dask/dask/blob/master/dask/array/core.py#L962-L967